java/ql/src/semmle/code/java/security/Encryption.qll

```diff
@@ -29,13 +29,20 @@ class SSLContext extends RefType {
   SSLContext() { hasQualifiedName("javax.net.ssl", "SSLContext") }
 }
 
+class SSLSession extends RefType {
```

Comment:
These additions add to the public api of `Encryption.qll` so they will need qldoc (we aim to have qldoc on all public classes and predicates in the libraries, but it's an on-going effort so we're not quite there yet, but CI will enforce it on newly added code).

Reply:
Done!